experimental/packages/opentelemetry-sdk-metrics-base/src/export/MetricExporter.ts

```diff
@@ -16,40 +16,38 @@
 
 import { AggregationTemporality } from './AggregationTemporality';
 import { MetricData } from './MetricData';
+import {
+  ExportResult,
+  ExportResultCode,
+} from '@opentelemetry/core';
 
 
 // https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/sdk.md#metricexporter
 
-// TODO should this just be an interface and exporters can implement their own shutdown?
-export abstract class MetricExporter {
+export abstract class PushMetricExporter {
```

Comment:
I'd think it would be better to define the PushMetricExporter as an interface:

1.  most of its methods are abstract anyway. 
2. keep it aligned with SpanExporter.

Reply:
Sounds good to me